success-page.php

```diff
@@ -0,0 +1,7 @@
+<?php include 'includes/header.php'; ?>
+<h1>SUCCESS!</h1>
+    <div class="thank-you-message">
+        <h2>Thank you <span class="first-name"></span>!</h2>
+        <p>We will respond to your query within 24 hours on <span class="email"></span>.</p>
+    </div> <!-- End of thank-you-message -->
+
```

Comment:
Add closing tag for container, body and html.

Reply:
done